ui/mirage/config.js

```diff
@@ -29,6 +31,33 @@ export default function() {
     };
   });
 
+  this.get('/sys/license/status', function() {
+    return {
+      autoloading_used: false,
+      stored: {
+        expiration_time: EXPIRY_DATE,
+        features: ['DR Replication', 'Namespaces', 'Lease Count Quotas', 'Automated Snapshots'],
+        license_id: '0eca7ef8-ebc0-f875-315e-3cc94a7870cf',
+        performance_standby_count: 0,
+        start_time: '2020-04-28T00:00:00Z',
+      },
+      // persisted_autoload: {
```

Comment:
curious why these are commented out?

Reply:
I was testing with various configurations but wanted to make sure we didn't lose the other options